ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -55,6 +55,11 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.23-openshift-4.19
   to: hypershift-tests
+- dockerfile_literal: |
+    FROM base
+    RUN dnf install -y python3-pip
```

Comment:
Originally I was going to install codespell here too but I think we want to maintain the specific codespell version we are using.